misc/python/materialize/checks/checks.py

```diff
@@ -26,6 +25,15 @@ def __init__(self, base_version: MzVersion, rng: Random | None) -> None:
     def _can_run(self, e: Executor) -> bool:
         return True
 
+    def _is_enabled(self) -> tuple[bool, str | None]:
```

Comment:
I kept it simple; alternatively, I could also introduce something like `YesIgnore(reason)` and `NoIgnore()` as in the output-consistency test framework.

Reply:
Does this work? tuple[False, str] | True

Reply:
Interesting... 😀 Let me give it a try!

Reply:
Nope: `Expected type expression but received "Literal[False]"`